bin/tronview

```diff
@@ -95,16 +96,26 @@ def view_job(args, job_id, client):
 
 
 def view_job_run(args, job_run_id, client):
-    actions = client.job_runs(job_run_id.url)
+    try:
+        actions = client.job_runs(job_run_id.url)
+    except RequestError as e:
+        response = e.args[1]
+        raise SystemExit(f"Error: {response.content}")
```

Comment:
In other words, I don't think it is "safe" for this function to "know" that we are going to be getting an object like this that is safe to inspect. 

If you would like to read another opinion on handling exceptions like this, you might like (the whole blog post, but this part in particular) 
https://blog.philipphauer.de/checked-exceptions-are-evil/#exposing-implementation-details-in-the-signature